airflow/kubernetes/pod_generator.py

```diff
@@ -351,7 +351,7 @@ def construct_pod(  # pylint: disable=too-many-arguments
         try_number: int,
         kube_image: str,
         date: datetime.datetime,
-        command: List[str],
+        args: List[str],
```

Comment:
I think pod generator should have options for command and args.

Reply:
The PodGenerator is only used internally by the KubernetesExecutor, we don't discuss it anywhere in the airflow documentation. It's not advertised as a part of the Airflow API.

Reply:
Oh, it's not used in mutation hook anymore. Cool, this is fine then